src/app/video-courses/highlight-course.directive.ts

```diff
@@ -0,0 +1,37 @@
+import { Directive, ElementRef, Input, OnInit } from '@angular/core';
+
+@Directive({
+  selector: '[appHighlightCourse]'
+})
+export class HighlightCourseDirective implements OnInit {
+  @Input('appHighlightCourse') courseDate: Date;
+
+  private static isFreshCourse(courseDate: Date, currentDate: Date) {
+    const millisecondsInDay = 1000 * 60 * 60 * 24;
+    const twoWeeks = 14;
+    const diffInDays = (currentDate.valueOf() - courseDate.valueOf()) / millisecondsInDay;
+
+    return diffInDays <= twoWeeks;
+  }
+
+  constructor(private el: ElementRef) {
+  }
+
+  ngOnInit() {
+    const currentDate = new Date();
+
+    if (currentDate < this.courseDate) {
```

Comment:
wrong comparison, use `currentDate.getTime() < this.courseDate.getTime()`

Reply:
Done